src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <mapped-superclass name="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroup"
+                       repository-class="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupRepository"
+                       table="at_target_groups">
+
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="title" column="title" type="string" length="255" nullable="false"/>
+        <field name="description" column="description" type="text" nullable="true"/>
+        <field name="priority" column="priority" type="integer" nullable="false"/>
+        <field name="allWebspaces" column="allWebspaces" type="boolean" nullable="false"/>
+        <field name="active" column="active" type="boolean" nullable="false"/>
+
+        <one-to-many field="webspaces"
+                     mapped-by="targetGroup"
+                     target-entity="Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroupWebspace">
+            <cascade>
+                <cascade-all/>
```

Comment:
no, TargetGroupWebspaces entities cannot be used for other target groups, since this is a one-to-many relationship:
<img width="833" alt="bildschirmfoto 2017-03-16 um 08 16 18" src="https://cloud.githubusercontent.com/assets/5152633/23985721/dddf29d8-0a20-11e7-9e33-10aa4e65072c.png">

So every entity should be persisted on it's own when created? This would mean i need to inject the entity manager into the service that creates the entities (e.g. Mapper Class or JmsSerializer Handler)